woocommerce/woocommerce-android
# Store creation: Plan purchase

This PR implements the plan purchase step of the store creation.

<img src="https://user-images.githubusercontent.com/1522856/201388144-55ca7021-8606-43b8-852c-1455450d1717.png" width="300" />

**To test:**
1. Start the store creation flow
2. When you get to the domain selection and tap on Continue, verify the plan purchase screen is shown
3. Tapping on the X button goes back to the previous screen
